src/Form/FieldCheckbox.js

```diff
@@ -10,91 +10,123 @@ export default class FieldCheckbox extends Util.mixin(BindMixin) {
     return ['handleChange'];
   }
 
-  componentDidMount() {
-    if (this.props.startValue === 'indeterminate') {
-      React.findDOMNode(this.refs.checkbox).indeterminate = true;
-    }
+  handleChange(name, modelState) {
+    let {props} = this;
```

Comment:
Yeah, @kennyt actually made me realize that this was a good use of the "properties spread" (can't remember its actual name)